hopsworks-web/yo/app/nav.html

```diff
@@ -1,88 +1,87 @@
+          <button type="button" class="hamburger is-open" style="color:white;" data-toggle="offcanvas" ng-click="projectCtrl.isClosed = !projectCtrl.isClosed">
             <span class="hamb-top"></span>
             <span class="hamb-middle"></span>
             <span class="hamb-bottom"></span>
           </button>
         </div>
       </div>
-
-      <div class="uib-collapse navbar-collapse" id="navbar">
-        <ul class="nav navbar-nav navbar-right"
-            style="margin-right:10px;">
-          <li ng-click="mainCtrl.getMessages()" uib-dropdown>
-            <a uib-dropdown-toggle style="padding:5px 15px 0px 10px; margin-top: 13px; margin-bottom: -15px;" aria-expanded="false">
+    </div>
+      <div class="collapse navbar-collapse" id="navbar">
+        <ul class="nav navbar-nav navbar-right" style="margin-right:10px;">
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" style="padding:7px 10px 8px 10px; text-decoration:none;"  aria-expanded="false" id="navbarProfile">
+              <img alt="" src="https://gravatar.com/avatar/{{mainCtrl.emailHash}}?d=retro&s=50" style="height: 35px; margin-right: 10px;border: 0.5px solid gray;" class="profile-image img-circle">
+              {{mainCtrl.email}} <b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              <li ng-show="mainCtrl.isAdmin() === 'true'"><a ng-click="mainCtrl.goToAdminPage()"><i class="fa fa-user-secret"></i>Admin</a></li>
+              <li><a ng-click="mainCtrl.profileModal()"><i class="fa fa-cog"></i> Account</a></li>
+              <!--<li><a ng-click="mainCtrl.sshKeysModal()"><i class="fa fa-key"></i> Ssh Keys</a></li>-->
+              <li><a href="#!/" ng-click="mainCtrl.logout()"><i class="fa fa-sign-out"></i> Sign out</a></li>
+            </ul>
+          </li>
+        </ul>
+        <ul class="nav navbar-nav navbar-right" style="margin-right:10px;">
+          <li class="dropdown" ng-click="mainCtrl.getMessages()">
+            <a class="dropdown-toggle" data-toggle="dropdown" style="padding:5px 15px 0px 20px; margin-top: 13px;" aria-expanded="false">
               <i ng-if="mainCtrl.unreadMessages" class="fa fa-envelope fa-lg" title="Unread messages">
                 <span class="badge" style="background-color: #f89406; font-family: sans-serif; z-index: 100; position: absolute; top: -10px; right: -18px">{{mainCtrl.unreadMessages}}</span>
               </i>
               <i ng-if="!mainCtrl.unreadMessages" class="fa fa-envelope-o fa-lg" title="No unread message"></i>
             </a>
-            <ul uib-dropdown-menu class="dropdown-menu dropdown-messages" >
-              <h3 ng-if="mainCtrl.messages.length" class="text-muted"> Latest messages</h3>
-              <h3 ng-if="!mainCtrl.messages.length" class="text-muted"> Inbox empty.</h3>
-              <li ng-repeat="msg in mainCtrl.messages | limitTo:5">
+            <ul class="dropdown-menu dropdown-messages" >
```

Comment:
I thought that "dropdown-menu" was all gone now with bootstrap 4. It should be uib-dropdown-menu. Are you sure about this? I remember changing all "dropdown-menu" to uib-dropdown when we upgraded to angular 1.6.

Reply:
It is still there in bootstrap 4, but we are using bootstrap 3.3.7. And uib is the angular version of bootstrap and we do not need to use it here. When we upgrade to angularjs 1.6 we changed <li class="uib-dropdown"> to <li uib-dropdown>